tools/tags/tag-browser.css

```diff
@@ -6,8 +6,28 @@
   display: block;
 }
 
+.da-tag-browser {
```

Comment:
Do we really want to be prefixing all of the classes with `da-`? Not sure that's a question you can answer, but I think it's worth discussing.

Reply:
That's a good point, since these plugins are iframes with web component any prefixing or name spacing would mostly be for readability.